Tests/Functional/PhpcrRepositoryTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ResourceBundle\Tests\Functional;
+
+class PhpcrRepositoryTest extends PhpcrRepositoryTestCase
+{
+    const REPOSITORY = 'test_repository_phpcr';
+
+    /**
+     * @dataProvider provideGet
+     */
+    public function testRepositoryGet($path, $expectedName)
+    {
+        $repository = $this->repositoryRegistry->get(self::REPOSITORY);
+        $res = $repository->get($path);
+        $this->assertNotNull($res);
+        $node = $res->getPayload();
+
+        $this->assertEquals($expectedName, $node->getName());
+    }
+
+    /**
+     * @dataProvider provideFind
+     */
+    public function testRepositoryFind($pattern, $nbResults)
+    {
+        $repository = $this->repositoryRegistry->get(self::REPOSITORY);
+        $res = $repository->find($pattern);
+        $this->assertCount($nbResults, $res);
+    }
+
+    /**
+     * @dataProvider provideMove
+     */
+    public function testRepositoryMove($sourcePath, $targetPath, $expectedNodeName)
+    {
+        $repository = $this->repositoryRegistry->get(self::REPOSITORY);
+        $res = $repository->move($sourcePath, $targetPath);
+
+        $this->assertEquals(1, $res);
+        $fooBar = $this->dm->find(null, '/test'.$targetPath);
+        $this->assertEquals($expectedNodeName, $fooBar->getNodeName());
+    }
+
+    /**
+     * @dataProvider provideDelete
+     */
+    public function testRepositoryRemove($path, $expectedDeleted)
+    {
+        $repository = $this->repositoryRegistry->get(self::REPOSITORY);
+        $res = $repository->remove($path);
+
+        $this->assertEquals($expectedDeleted, $res);
+        $this->assertNull($this->dm->find(null, $path));
+    }
+
+    /**
+     * @dataProvider provideAdd
+     */
+    public function testRepositoryAddSingleResource($path, $name)
+    {
+        $this->markTestSkipped('Not able to test that atm.');
+    }
+
+    /**
+     * @dataProvider provideAdd
+     */
+    public function testRepositoryAddResourceCollection($path, $name)
+    {
+        $this->markTestSkipped('Not able to test that atm.');
```

Comment:
You can only create PHPCR nodes through the PHPCR session with `$node = $session->getNode('foobar')->addNode('bar');` iirc.